cmd/minikube/cmd/config/addons_list.go

```diff
@@ -44,17 +49,38 @@ var addonsListCmd = &cobra.Command{
 		if len(args) != 0 {
 			exit.UsageT("usage: minikube addons list")
 		}
-		err := addonList()
-		if err != nil {
-			exit.WithError("addon list failed", err)
+
+		if addonListOutput != "list" && addonListFormat != defaultAddonListFormat {
+			exit.UsageT("Cannot use both --output and --format options")
+		}
+
+		switch strings.ToLower(addonListOutput) {
+		case "list":
+			PrintAddonsListList()
```

Comment:
Generally stuttering like ListList() is discouraged in Go, just `PrintAddonsList()` should be enough here.